### ISSUE TITLE
Real usefulness of F6 as "QSO B4" and replacement by "NR?"

### ISSUE THREAD

**Issue by f6fvy** [w7sst/MorseRunner#274]
## Summary

Modify the "QSO B4" message by a "NR?" message and process it to repeat the exchange only (once or twice, randomly)

## Motivation

In real life, you never send a "QSO B4". Even if you're sure you're being called by a dupe (which you can't check in Morse Runner), you'll just quietly redo the QSO. It's much quicker than explaining, and in most cases the caller misunderstood your callsign the first time.

That's why I suggest deleting this case, and replacing it instead with an "NR?" to ask to repeat only the exchange sent. This is different from F7 ("?"), which returns everything (callsign + exchange).

## Can you help?

Please let us know if you are available to help. (replace '[ ]' with '[x]' to affirm)
- [X] Yes, I'm available to discuss possible solutions and alternatives to this problem.
- [X] Yes, I'd like to review and provide feedback on the proposed solution.
- [X] Yes, I'm available to help test a solution to this problem.

**Comment by w7sst** [comment]
Good idea. I notice in the existing `Run` menu the `F12` key is assigned as sending `NR?`. Does this provide the behavior you are looking for. Perhaps some of the code for your proposed change is already written.

**Comment by f6fvy** [comment]
Good catch. I never noticed there was a process attached to F12 ;-)  TBH, I rarely use this menu, only the buttons on right pane.

**Comment by f6fvy** [comment]
And what about a "CL?" message to ask for a callsign repeat instead of F8 "NIL" ? Useful if you correctly got the exchange and have a doubt on the callsign.
